README.md

```diff
@@ -1 +1,144 @@
-# Bookstore-App
\ No newline at end of file
+<a name="readme-top"></a>
+
+# Math Magician Project
+
+# 📗 Table of Contents
+
+- [Math Magician Project](#math-magician-project)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Math Magician Project ](#-math-magician-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites ](#prerequisites-)
+    - [Setup ](#setup-)
+    - [Install ](#install-)
+    - [Usage ](#usage-)
+    - [Run tests ](#run-tests-)
+  - [👥 Authors ](#-authors-)
+    - [👤 **Mert Kantaroglu**](#-mert-kantaroglu)
+    - [👤 **Josue Rushanika**](#-josue-rushanika)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Math Magician Project <a name="about-project"></a>
+
+**Math Magician is a SPA project which you can make single calculations and reading a random math related quote**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **Homepage**
+- **Calculator**
+- **Quote**
```

Comment:
- [ ] I did not see a calculator or quote in this app, maths magician and book store apps are two different apps, kindly make corrections to make the app look professional.